features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,189 +1,651 @@
-      | vrouw    | man      | Ridder   | Partner na eigen      | W. van der Veen-van den Aedel           | mevrouw Van der Veen-van den Aedel         |
-      | vrouw    | man      | Jonkheer | Eigen                 | A.C. van der Veen                       | mevrouw Van der Veen                       |
-      | vrouw    | man      | Jonkheer | Partner na eigen      | A.C. van der Veen-van den Aedel         | mevrouw Van der Veen-van den Aedel         |
-      | vrouw    | man      | Jonkheer | Partner               | A.C. van den Aedel                      | mevrouw Van den Aedel                      |
-      | vrouw    | man      | Jonkheer | Partner voor eigen    | A.C. van den Aedel-van der Veen         | mevrouw Van den Aedel-van der Veen         |
+  De gebruikInLopendeTekst bij een persoon wordt gevuld door de provider om op deze wijze op eenduidige wijze een persoon te kunnen benoemen.
+
+  De gebruikInLopendeTekst kan worden gebruikt in bijvoorbeeld een zin in een brief als "In uw brief van 12 mei jongstleden, geeft u het overlijden van uw vader, de heer in het Veld, aan.", waarbij "de heer in het Veld" gehaald is uit attribuut gebruikInLopendeTekst.
+
+  # In onderstaande tabellen betekenen de afkortingen:
+  # GA = "mevrouw", "de heer" of de voorletters
+  # VL = voorletters
+  # VV = voorvoegselGeslachtsnaam
+  # GN = geslachtsnaam
+  # VP = voorvoegselGeslachtsnaam partner
+  # GP = geslachtsnaam partner
+  # AT = adelijke titel
+  # PK = predikaat
+  # HT = hoffelijkheidstitel of 'titre de courtoisie'
+
+Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predikaat wordt samengesteld afhankelijk van de geslachtsaanduiding en de waarde van aanduidingNaamgebruik, waarbij geldt dat:
+  - gebruikInLopendeTekst wordt voorafgegaan door "mevrouw", "de heer" of de voorletters, afhankelijk van de geslachtsaanduiding:
+    | geslachtsaanduiding | gebruikInLopendeTekst begint met (GA) |
+    | M                   | de heer                               |
+    | V                   | mevrouw                               |
+    | O                   | VL                                    |
+  - gebruikInLopendeTekst wordt samengesteld uit naamcomponenten van de persoon en eventuele (ex)partner op basis van de aanduidingNaamgebruik:
+    | aanduidingNaamgebruik | gebruikinlopendetekst |
+    | E                     | GA VV GN              |
+    | P                     | GA VP GP              |
+    | V                     | GA VP GP-VV GN        |
+    | N                     | GA VV GN-VP GP        |
+  - Een voorvoegsel wordt met kleine letters wordt geschreven
+  - De eerste naamcomponent in gebruikInLopendeTekst anders dan "mevrouw" of "de heer" begint met een hoofdletter
+  - Wanneer voorletters zijn opgenomen in de gebruikinlopendetekst, het voorvoegsel van de eerste naam in de gebruikinlopendetekst met alleen kleine letters wordt geschreven
```

Comment:
Deze zin loopt niet lekker waardoor ik het niet begrijp

Reply:
ik wijzig het naar de manier waarop het in de volgende rule (persoon met adellijke titel) staat beschreven: "De eerste naamcomponent direct na "mevrouw" of "de heer" begint met een hoofdletter"